webpack.webext.babel.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+import config, { src, dist } from './webpack.common';
+import pkg from './package.json';
+
+const variant = process.env.VARIANT; // 'chrome', 'firefox' or 'opera'
+
+// Configure separate entry points.
+
+config.entry('popup').add(src.webext('popup', 'popup.jsx'));
+config.entry('content').add(src.webext('content.js'));
+config.entry('background').add(src.webext('background.js'));
+
+// Set browser-specific output path.
+
+config.output.path(dist(variant));
+
+// Copy the manifest.json template in addition to default files.
+
+config.plugin('copy').tap(([patterns]) => [[
+  ...patterns,
+  {
+    from: src.webext('manifest.json'),
+    transform: (content) => {
+      const mf = JSON.parse(content);
+
+      mf.name = pkg.name;
+      mf.version = pkg.version;
+      mf.description = pkg.description;
+
+      if (variant === 'firefox') {
+        mf.applications = {
+          gecko: {
+            id: 'tickety-tick@bitcrowd.net',
+          },
```

Comment:
we only need the addon id on firefox if we want to manually install it. 
I think it could cause trouble in the version that we release in the store (if I remember correctly)

Reply:
Aaah. Now I am confused 😕 😁 

I remember that in #54, we added the "applications" entry because there were issues when trying to install a locally built version in the Firefox Quantum preview.

However, this [Mozilla page](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID#When_do_you_need_an_Add-on_ID) says we do not need it all and I just tried it with the official Firefox Quantum 57.0 and it worked without complaints.

Can you please give it a try with these lines commented out @andreasknoepfle?

If successful, I'd say we remove those lines again.

Reply:
<img width="1026" alt="…installed without gecko ID" src="https://user-images.githubusercontent.com/706519/33096964-a0ef94fe-cf00-11e7-870a-a7d9fe4aab0f.png">

Reply:
### On Firefox:
You can: 
* Install it manually, temporarily with or without Addon-ID
* Install it manually, permanently with Addon-ID
* Install it from the marketplace, permanently without Addon-ID

There are errors when: 
* Installing manually, permanently without Addon-ID
* Releasing a version to the marketplace with a self-chosen Addon-ID 

### On Chrome:
You can:
* Do anything without an Addon-ID

Get warnings:
* with anything with an Addon-ID

Reply:
So the question is, do we want to keep the ability to manually install the addon, when firefox is not publishing it for quite a while again.

Reply:
🤦‍♂️ Oh, I see. Thanks for figuring that out… 👍

Reply:
@andreasknoepfle: What do you think about separate `build:firefox` and `build:firefox-local` scripts where only the latter inserts the addon-ID?

Reply:
👍, that would do the trick